aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,32 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## October 25 - November 5, 2021
+
+### Hacktoberfest 
+- Grommet completed the month-long [Hacktoberfest event](https://grommet.slack.com/archives/C3UGPN06S/p1636133843018400) 
+which encourages support and contributions to open source projects. This year 79 pull 
+requests were merged addressing TypeScript support and type testing, support for Storybook 
+theming, and a smattering of bug fixes and other enhancements.
+
+### "Big Things" progress updates
+Here are the latest developments regarding roadmap items:
+- **NameValueList**. Styling for NameValueList and NameValuePairs are now available in the 
+HPE Theme.
```

Comment:
Theres actually  nothing in hpe theme just all in base.js

Reply:
It is available though ;).

Reply:
@halocline I just added my ticket to #1990 to done which means the name-value list component and variants are published to the HPE Design System Figma library.

Reply:
Would this be appropriate to include in an update issued at the completion of this sprint (i.e. Monday morning)?

My thinking is that it could be nice and clean to simply run through the items in the Done column at the completion of the sprint.

For this draft, I only included items completed in the sprints w/o 10/25 - 10/29 & w/o 11/1 - 11/5.

Reply:
Sounds good to me! Thanks for clarifying :)